tools/sidekick/config.json

```diff
@@ -1,15 +1,25 @@
 {
-    "project": "Boilerplate",
-    "plugins": [
-        {
-            "id": "cif",
-            "title": "Commerce",
-            "environments": [
-                "edit"
-            ],
-            "url": "https://main--aem-boilerplate-commerce--hlxsites.hlx.live/tools/picker/dist/index.html",
-            "isPalette": true,
-            "paletteRect": "top: 54px; left: 5px; bottom: 5px; width: 300px; height: calc(100% - 59px); border-radius: var(--hlx-sk-button-border-radius); overflow: hidden; resize: horizontal;"
-        }
-    ]
-}
\ No newline at end of file
+  "project": "Boilerplate",
+  "plugins": [
+    {
+      "id": "cif",
+      "title": "Commerce",
+      "environments": [
+        "edit"
+      ],
+      "url": "https://main--aem-boilerplate-commerce--hlxsites.hlx.live/tools/picker/dist/index.html",
```

Comment:
I would have suggested you use `aem.live` urls in this file

Reply:
Hi @sirugh , that was the original url I have copied from existing `Commerce` plugin - should I change it in both plugins?